python/core/auto_generated/qgsbasemappathregistry.sip.in

```diff
@@ -0,0 +1,75 @@
+/************************************************************************
+ * This file has been generated automatically from                      *
+ *                                                                      *
+ * src/core/qgsbasemappathregistry.h                                    *
```

Comment:
As I was hinting in the comment yesterday, I feel BaseMap isn't really the best wording representation of what this feature does. While I'm sure most use cases will be to share those heavy raster 2gb topo maps, in theory, this can be used for anything really (from all types of layers, assets like logos, etc.)

Alternatives I thought of overnight:
- DataBank
- DataRepository